setup.py

```diff
@@ -40,7 +40,8 @@ def read(fname):
         'schema',
         'PythonWebHDFS',
         'colorama',
-        'scikit-learn>=0.20,<0.22'
+        'scikit-learn>=0.20,<0.22',
+        'pkginfo'
```

Comment:
also need to add this package in /deployment/pypy/setup.py?

Reply:
Thanks, added.